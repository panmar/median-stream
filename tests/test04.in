84283753
444324288
-521522403
216238976
-1195866396
1612821088
1312218194
2014335892
-510856313
1570681362
