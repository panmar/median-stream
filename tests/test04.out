84283753
264304020.5
84283753
150261364.5
84283753
150261364.5
216238976
330281632
216238976
330281632
